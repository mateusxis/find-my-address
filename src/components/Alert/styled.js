import styled from 'styled-components';

export const SAlert = styled.label`
  display: block;
  text-align: center;
  margin: 0;
  padding: 0.8rem 1.6rem;
  color: inherit;
  width: 80%;
  font-family: inherit;
  font-size: 1.5rem;
  font-weight: inherit;
  line-height: 1.5;
  border: none;
  border-radius: 0.4rem;
  transition: box-shadow 300ms;
  color: #455a64;
  background-color: #ffffff;

  @media (min-width: 601px) {
    font-size: 1.8rem;
  }

  @media (min-width: 961px) {
    font-size: 2.1rem;
  }
`;
