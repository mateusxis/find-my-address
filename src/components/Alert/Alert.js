import { SAlert } from './styled';

const Alert = () => <SAlert>CEP buscado n√£o encontrado ou incorreto.</SAlert>;

export default Alert;
