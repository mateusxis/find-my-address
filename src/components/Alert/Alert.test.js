import renderer from 'react-test-renderer';

import Alert from './Alert';

describe('Alert', () => {
  it('should render Alert', () => {
    const tree = renderer.create(<Alert />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
