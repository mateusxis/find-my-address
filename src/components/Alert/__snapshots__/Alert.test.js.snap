// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render Alert 1`] = `
<label
  className="sc-bdvvtL fZpmBV"
>
  CEP buscado n√£o encontrado ou incorreto.
</label>
`;
